bin/run-bisect

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+# For this to work properly, your configuration file must set
+#   USE_CLUSTER_SPARK = False
+#   SPARK_COMMIT_ID = os.environ["SPARK_COMMIT_ID"]
+
+FWDIR="$(cd "`dirname "$0"`/../"; pwd)"
+
+if [ "$#" -ne 4 ]; then
+  echo "Test every Nth commit between two commits"
+  echo "Usage: oldest_commit newest_commit N config_file"
+  exit 1
+fi
+
+oldestCommit="$1"
+newestCommit="$2"
+NR="$3"
+configFile="$4"
+
+# Find the SHAs of every NRth commit between two git commits:
+cd "$FWDIR/spark-build-cache/master"
+git fetch
+versions=($(git log --oneline "$oldestCommit..$newestCommit" | awk "NR == 0 || NR % $NR == 0" | cut -d ' ' -f1))
```

Comment:
I think there is a minor problem with this line. It outputs the intermediate commits as short hashes, which in turn affects how the logs and results are named. This is inconsistent with how `run-tests` does things, which is use the full commit hash.